src/utils.ts

```
@@ -17,7 +17,7 @@ export const timeUnits = {
 } as const;
 
 const units = values(timeUnits);
-export type TimeUnit = (typeof units)[number];
+export type TimeUnit = typeof units[number];
```

ESLint

[error] Replace `typeof·units` with `(typeof·units)`

(prettier/prettier)